randovania/games/fusion/logic_database/Sector 3 PYR.txt

```diff
@@ -350,7 +350,10 @@ Extra - room_id: [8]
           Morph Ball and Screw Attack and Knowledge (Beginner)
           All of the following:
               Can Use Bombs
-              Mid-Air Morph (Intermediate) or Can Use Springball
+              Any of the following:
+                  Can Use Springball
+                  # https://youtu.be/UBvgvEpVc-Q
```

Comment:
```suggestion
                  # Mid Air Morph: https://youtu.be/UBvgvEpVc-Q
```

Reply:
kinda seems redundant to put mid-air morph as a comment for a trick that is called mid-air morph but I don't care enough to argue